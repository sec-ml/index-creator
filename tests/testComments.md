| term                                                       | sub-term                           | notes                   | book | page |
| ---------------------------------------------------------- | ---------------------------------- | ----------------------- | ---- | ---- |
| ?This whole row should be hidden thanks to comment symbol! | grep && cat && tail && cd && mkdir | linux built-in commands | 1    | 3    |
| normal row                                                 | 2                                  | 3                       | 4    | 5    |
| ? this comment row "contains"^"contains long expansions"   | yep!                               | it does                 | 7    | 8    |
| contains                                                   | contains                           | contains                | 6    | 5-7  |
