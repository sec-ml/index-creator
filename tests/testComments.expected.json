[
  {
    "term": "<span class=\"abbr-contains\">contains long expansions</span>",
    "subTerm": "<span class=\"abbr-contains\">contains long expansions</span>",
    "notes": "<span class=\"abbr-contains\">contains long expansions</span>",
    "book": 6,
    "page": "5-7"
  },
  {
    "term": "normal row",
    "subTerm": "2",
    "notes": "3",
    "book": 4,
    "page": "5"
  }
]
